src/components/functional/Chart.jsx

```diff
@@ -0,0 +1,149 @@
+"use strict";
```

Comment:
You don't need this as ES6 modules [are always strict mode](https://tc39.github.io/ecma262/#sec-strict-mode-code).